docs/BR.md

```diff
@@ -3077,7 +3058,7 @@ Table: Encoding and Order Requirements for Selected Attributes
 | `organizationalUnitName` | `2.5.4.11` | [RFC 5280](https://tools.ietf.org/html/rfc5280) | MUST use `UTF8String` or `PrintableString` | 32 |
```

Comment:
Is the max length of the OU field 32 or 64 characters? RFC5280 defines ub-organizational-unit-name INTEGER ::= 64